docs/translation/index.md

```diff
@@ -99,7 +99,29 @@ The general rules for avoiding errors may be summed up like this:
     ```smarty
     <p>{ts}Hello, world!{/ts}</p>
     ```
-    
+
+For `title` attributes in `<a>` links, within CiviCRM these usually only appear in links that aren't within a larger block of text or where there is no clickable text, such as a datepicker icon. In this situation, the title text needs to be translated:
+
+!!! failure "Bad"
+
+    ```smarty
+    {ts}<a href="https://www.example.org/civicrm/contact/view?cid=1" title="Go to the contact page">View Contact</a>{/ts}
+    ```
+
+!!! failure "Less bad"
+
+    ```smarty
+    <a href="https://www.example.org/civicrm/contact/view?cid=1" title="{ts}Go to the contact page{/ts}">{ts}View Contact{/ts}</a>
+    ```
```

Comment:
I'm not sure I get why this is still sorta bad. Because the title & link text are redundant?

Reply:
Yes - I think it needs a better example perhaps?

Reply:
I was basing the example on existing defacto usage and I'm not clear myself whether there is a "good" way to do title attributes. I put "less bad" because you don't have the context that they go together.

Reply:
Updated example to one where the title text is more different from the link text.